tjmonopix2/system/tjmonopix2.py

```diff
@@ -581,6 +600,12 @@ def make_mask_for_step(self, step):
 
 
 class TJMonoPix2(object):
+    """Base class for TJ-Monopix functionality
+
+    Parameters
+    ----------
+    object : class
+    """
```

Comment:
Functionality is a bit too general. "For operation of TJ-Monopix2. One instance corresponds to a connected chip".